common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -235,14 +281,58 @@ public ICacheRecord save(@NonNull final GenericOAuth2Strategy oAuth2Strategy,
         result.setAccount(accountToSave);
         result.setAccessToken(accessTokenToSave);
         result.setRefreshToken(refreshTokenToSave);
+        setToCacheRecord(result, idTokenToSave);
 
-        if (CredentialType.V1IdToken.name().equalsIgnoreCase(idTokenToSave.getCredentialType())) {
-            result.setV1IdToken(idTokenToSave);
-        } else {
-            result.setIdToken(idTokenToSave);
+        return result;
+    }
+
+    @Override
+    @NonNull
+    public List<ICacheRecord> saveAndLoadAggregatedAccountData(
+            @NonNull final GenericOAuth2Strategy oAuth2Strategy,
+            @NonNull final GenericAuthorizationRequest request,
+            @NonNull final GenericTokenResponse response) throws ClientException {
+        return mergeCacheRecordWithCorollaryAccounts(
+                save(oAuth2Strategy, request, response)
+        );
+    }
+
+    /**
+     * Given an AccountRecord and associated client_id, load a sparse ICacheRecord containing
+     * the provided AccountRecord and its accompanying IdTokens.
+     *
+     * @param clientId The client_id relative to which IdTokens should be loaded.
+     * @param acct     The target AccountRecord.
+     * @return A sparse ICacheRecord containing the provided AccountRecord and its IdTokens.
+     */
+    ICacheRecord getSparseCacheRecordForAccount(@NonNull final String clientId,
```

Comment:
Not sure I followed what "sparse" was meant to convey here.

Reply:
I added a clarifying sentence to the javadoc in 1cebf6e0deab478263b0e35a0fd6ef34218956b7 - I am borrowing sparse here to signal that elements inside this structure are going to be `null`